### DISCUSSION TITLE
Hangs at boot

### DISCUSSION THREAD

**Discussion by oliviercailloux** [ipxe/ipxe#932] [Q&A]
I successfully chainloaded iPXE (I can boot the iPXE PHP demo) but I can’t boot WinPE. Everything seems to work as expected until it says “Entering bootmgr.exe with parameters at 0x2dfc0”, and it hangs. The Apache server logs show that the file boot.wim is asked and served to the client.

I have no idea what to try to debug the issue.

Here is the config I tried most.

```
#!ipxe

cpuid --ext 29 && set arch amd64 || set arch x86
kernel wimboot
initrd media/sources/boot.wim  boot.wim
boot
```

Here are some screenshots with an alternative config. This sample uses supplementary keywords “rawbcd linear gui” and `imgstat` and `prompt` commands that I tried for lack of better idea; it hangs in the same way if I omit them.

![IMG_20230401_123446412](https://user-images.githubusercontent.com/1244880/229285156-cf3b3480-21eb-491b-97ee-c086648488ee.jpg)
![IMG_20230401_123458707](https://user-images.githubusercontent.com/1244880/229285192-35374cdd-8c89-4a50-82ab-14741c8e73b8.jpg)

Any idea to obtain information about what goes wrong would be appreciated!

**Comment by oliviercailloux** [comment]
… also wondering whether the messages "directory entry “DVD” not found" and "directory entry “sms” not found" are worrying?

**Comment by stappersg** [comment]
https://ipxe.org/wimboot#troubleshooting

**Comment by NiKiZe** [comment]
What is the source of boot.wim? Have you tried the unmodified boot.wim from iso?

Also please run an imgstat before your boot command. You might need an imgfree if something unexpected is loaded.

**Reply by oliviercailloux** [reply]
> What is the source of boot.wim? Have you tried the unmodified boot.wim from iso?

Unmodified from the ADK result of `copype amd64 C:\temp\winpe\amd64` (then moved to the webserver), as per iPXE’s [Network-booting Windows PE](https://ipxe.org/howto/winpe) doc.

> Also please run an imgstat before your boot command. You might need an imgfree if something unexpected is loaded.

The screenshot above shows the result of the `imgstat` command. Not sure if anything is “unexpected” there but it doesn’t seem so. Should it list something more?

**Reply by NiKiZe** [reply]
Yes sorry, nothing weird. Except for that you don't want rawbcd and linear here.
Just för sanity take boot.wim from a standard windows install iso.
